package.json

```diff
@@ -60,6 +60,8 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
+    "get-stream": "^6.0.1",
+    "git-log-parser": "^1.2.0",
```

Comment:
I think we could probably use `conventional-recommended-bump`, which is already pulled in as a dependency, and avoid pulling in new `get-stream` and `git-log-parser` deps.

Reply:
Quickly looking at the project, it seems to only expose a `releaseType`, do you think for a scenario where we only have commits with type `docs` , `conventional-recommended-bump` will return a `relaseType` of `null/undefined` opposed to `patch`?

Nonetheless, I will take a closer look into it!

Reply:
ok, I tested the following scenarios
having the following commit history  examples:
```
docs: update README
(tag: v9.2.0) chore: release 9.2.0 (#736)
```
running `conventional-recommended-bump -p angular` would result in `patch`

```
feat: added cool stuff
docs: update README
(tag: v9.2.0) chore: release 9.2.0 (#736)
```
running `conventional-recommended-bump -p angular` would result in `minor`

Reply:
Looking at options we might be able to take advantage of [whatBump](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-recommended-bump#whatbump)

What do you think of checking potential releases in the `whatBump` callback, the `commits` argument given to it seems to be equivalent information to what we get from `latest-commits.js`, so we would only need to check for release worthy commit and exit if none found (what we do with the `detectRelease` arg). 

https://github.com/conventional-changelog/standard-version/blob/280c4c0c073e220403814f94c79bbebb33a3c5f7/index.js#L47-L52

Reply:
@nelsonfncosta I like the idea of trying to use `whatBump`, we already do the parsing in the various parts of the conventional changelog repo, so I'm hesitant to pull in any other deps -- you might also be able to leverage some of the dependencies used within the conventional changelog monorepo already?

Reply:
@bcoe, can you take another look when possible 🙏 

I was able to use `whatBump` and avoid pulling new dependencies, I split the new implementation into a separate commit to be easier to compare and see what changed.